File: src/main/kotlin/com/hedvig/underwriter/config/Money.java
package com.hedvig.underwriter.config;

import com.fasterxml.jackson.databind.Module;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;
import org.zalando.jackson.datatype.money.MoneyModule;

@Configuration
public class Money {
  @Bean
  public Module monetaModule() {
    return new MoneyModule().withQuotedDecimalNumbers();
  }
}

File: src/main/kotlin/com/hedvig/underwriter/config/Money.kt
package com.hedvig.underwriter.config

import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import org.zalando.jackson.datatype.money.MoneyModule

@Configuration
class Money {
    @Bean
    fun monetaModule() =
        MoneyModule().withQuotedDecimalNumbers()
}
